package.json

```diff
@@ -60,6 +61,7 @@
   },
   "dependencies": {
     "core-js": "^1.0.0",
+    "babel-core": "^6.3.17",
```

Comment:
I'm going to followup and move this - it's not a dep of fbjs and really shouldn't even be a devdep (should be a devdep of fbjs-scripts, but we aren't running tests there right now so it's in the grey area)